src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -296,7 +314,10 @@ public ArrayList<String> getChildrenIds(String parentId) throws SQLException {
     public void clearTableRecords() throws SQLException {
```

Comment:
I know this is only used for testing, but consider updating the method name to something like clearTables as it now clears more than just the records table

Reply:
Yes, better name.

Reply:
Unit tests are failing because it hasn't been changed in: DsStorageUnitTestUtil.java line 57